delta/plugins/storage/src/test/resources/files/database/file-created.json

```diff
@@ -3,6 +3,7 @@
   "project" : "myorg/myproj",
   "storage" : "https://bluebrain.github.io/nexus/vocabulary/disk-storage?rev=1",
   "storageType" : "DiskStorage",
+  "tag" : null,
```

Comment:
For back-compat clarity it might be better to write nothing than to write `null`?